pkgs/data/fonts/tewi/default.nix

```diff
@@ -0,0 +1,47 @@
+{stdenv, fetchgit, bdftopcf, mkfontdir, mkfontscale}:
+
+stdenv.mkDerivation rec {
+  date = "2015-03-08";
+  name = "tewi-font-${date}";
+
+  src = fetchgit {
+    url = "https://github.com/lucy/tewi-font";
+    rev = "21a86aa5fd14f7d8ee2a4eab454cca21d11b61c1";
+    sha256 = "daa1d12f279814ce74ef92b38f8093df64f41206b5b3f1b924b2d48dec56b542";
+  };
+
+  buildInputs = [ bdftopcf mkfontdir mkfontscale ];
+  buildPhase = ''
+    for i in *.bdf; do
+        bdftopcf -o $(echo -n $i | sed 's/bdf/pcf/') $i
```

Comment:
Could replace `$(echo -n $i | sed 's/bdf/pcf/')` here by the more idiomatic `${i/bdf/pcf}` or `${i%.bdf}.pcf`.